File: app/src/main/java/com/parkingfinder/interfaces/OnItemClickedListener.java
package com.parkingfinder.interfaces;

import com.google.firebase.firestore.GeoPoint;
import com.parkingfinder.models.ParkingLot;

public interface OnItemClickedListener {
    void openMaps(GeoPoint location);
    void openParkingLotView(ParkingLot parkingLot);
}

File: app/src/main/java/com/parkingfinder/interfaces/OnItemClickedListener.kt
package com.parkingfinder.interfaces

import com.google.firebase.firestore.GeoPoint
import com.parkingfinder.models.ParkingLot

interface OnItemClickedListener {
    fun openMaps(location: GeoPoint?)
    fun openParkingLotView(parkingLot: ParkingLot?)
}